src/6_branch.js

```diff
@@ -1965,3 +1965,37 @@ Branch.prototype['trackCommerceEvent'] = wrap(callback_params.CALLBACK_ERR, func
 	});
 	done();
 });
+/*
+ logEvent() documentation
+ */
+Branch.prototype['logEvent'] = wrap(callback_params.CALLBACK_ERR, function(done) {
```

Comment:
Can't we just define all of the parameters now instead of using `arguments` to pull them out? Seems like something changed and now they are static